install/resources/settings.xml

```diff
@@ -1537,6 +1537,14 @@ min=1]]></optionscode>
 			<settingvalue><![CDATA[96]]></settingvalue>
 			<isdefault>1</isdefault>
 		</setting>
+		<setting name="attachimagestock">
+			<title>Stock Image Path</title>
+			<description><![CDATA[Select an stock image to use for image attachments when users have no permission to download.]]></description>
+			<disporder>6</disporder>
+			<optionscode><![CDATA[text]]></optionscode>
+			<settingvalue><![CDATA[images/attachment_stock.svg]]></settingvalue>
```

Comment:
We should probably include a basic default stock image with the core.

Reply:
Also again rename attachment_stock into attachment_default :)